client/src/components/Select/Select.stories.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import { Select } from "./Select";
+
+export default {
+  title: "Select",
```

Comment:
Should be under `Input/`.

I'm going to stop commenting on the actions thing, just keep it in mind and re-review things yourself after reading up on it :+1:

Reply:
Done!